documentation/build/html/_sources/antisphinx.txt

```diff
@@ -45,7 +45,6 @@ File Layout
 ***********
```

Comment:
Please investigate whether we can at least exclude the 'build/html/_sources' directory from version control, or whether the data therein is needed for showing the generated documentation.